driver-core/src/main/java/com/datastax/driver/core/policies/LoggingRetryPolicy.java

```diff
@@ -33,6 +35,31 @@
 public class LoggingRetryPolicy implements RetryPolicy {
 
     private static final Logger logger = LoggerFactory.getLogger(LoggingRetryPolicy.class);
+
+    @VisibleForTesting static final String IGNORING_READ_TIMEOUT =
+        "Ignoring read timeout (initial consistency: {}, required responses: {}, received responses: {}, data retrieved: {}, retries: {})";
+
+    @VisibleForTesting static final String RETRYING_ON_READ_TIMEOUT =
+        "Retrying on read timeout on {} at consistency {} (initial consistency: {}, required responses: {}, received responses: {}, data retrieved: {}, retries: {})";
+
+    @VisibleForTesting static final String IGNORING_WRITE_TIMEOUT =
+        "Ignoring write timeout (initial consistency: {}, write type: {}, required acknowledgments: {}, received acknowledgments: {}, retries: {})";
+
+    @VisibleForTesting static final String RETRYING_ON_WRITE_TIMEOUT =
+        "Retrying on write timeout on {} at consistency {} (initial consistency: {}, write type: {}, required acknowledgments: {}, received acknowledgments: {}, retries: {})";
+
+    @VisibleForTesting static final String IGNORING_UNAVAILABLE =
+        "Ignoring unavailable exception (initial consistency: {}, required replica: {}, alive replica: {}, retries: {})";
+
+    @VisibleForTesting static final String RETRYING_ON_UNAVAILABLE =
+        "Retrying on unavailable exception on {} at consistency {} (initial consistency: {}, required replica: {}, alive replica: {}, retries: {})";
+
+    @VisibleForTesting static final String IGNORING_UNEXPECTED_ERROR =
```

Comment:
I think these log messages should be updated to "Ignoring request error" / "Retrying on request error" to match the method name.